.gitlab/ci/.gitlab-ci.on-push.yml

```
@@ -416,6 +417,7 @@ slack-notify-nightly-build:
     - .push-rule
   variables:
     EXTRACT_PRIVATE_TESTDATA: "true"
+    BUILD_TYPE: "build"
```

I think this name is better since in the test upload we are replacing this global var, and it may be not easy to understand why.  (Change in all places)

```suggestion
    CLOUD_MACHINES_TYPE: "build"
```